src/popup/utils/utils.ts

```diff
@@ -443,11 +453,19 @@ export function getAccountNameToDisplay(acc: IAccount | undefined) {
   return acc?.name || `${i18n.t('pages.account.heading')} ${(acc?.idx || 0) + 1}`;
 }
 
-export function defaultTransactionSortingCallback(
-  a: ITransaction,
-  b: ITransaction,
+export function sortTransactionByDate(
```

Comment:
The function name is changed, but the logic not. So currently the name is not correct as this function is not actually sorting anything.

Reply:
check it now please

Reply:
I added new comment regarding the name so I'm closing this one.